src/schemas/22-10203/schema.js

```diff
@@ -95,7 +101,7 @@ const schema = {
     'degreeName',
     'schoolName',
     'schoolCity',
-    'schoolState',
+    'schoolCountry',
```

Comment:
Don't we still need schoolState?

Reply:
It's required in the UI if the country is USA, but it isn't _required_

Reply:
Doh! Gotcha